users/tests.py

```diff
@@ -11,13 +11,18 @@
 from plio.cache import get_cache_key
 
 
-class OtpAuthTestCase(BaseTestCase):
+class OtpAuthTestCase(BaseTestCase):  
     """Tests the OTP functionality."""
 
     def setUp(self):
         super().setUp()
         # unset client credentials token so that the subsequent API calls goes as guest
         self.client.credentials()
+    def test_request_otp_mobile_not_passed(self):
+        response = self.client.post(
+            reverse("request-otp")
+        )
+        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
```

Comment:
do check for the error message as well - the way Manya has done in her PR: https://github.com/avantifellows/plio-backend/pull/304

Reply:
corrected